README for V2.0 (#1492)
souffle-lang/souffle: README.md

# Welcome!

This is the official repository for the [Soufflé](https://souffle-lang.github.io) language project.  

> Soufflé language is similar to Datalog (but has terms known as records), and is frequently used as a 

Reviewer: ```suggestion
The Soufflé language is similar to Datalog (but has terms known as records), and is frequently used as a 
```

    
Build instructions can be found [here](https://souffle-lang.github.io/build).

## Legacy code

If you have written code for an older version of Souffle, please use the command line flag `--legacy`. 
Alternatively, please add the following line to the start of your source-code:
```
.pragma "legacy"
```

## How to contribute

Issues and bug reports for Souffle are found in the [issue list](https://github.com/souffle-lang/souffle/issues). 
This list is also where new contributors may find extensions / bug fixes to work on.

To contribute in this repo, please open a pull request from your fork of this repository.
The general workflow is as follows.

1. Find an issue in the issue list.
2. Fork the [souffle-lang/souffle](http://github.com/souffle-lang/souffle.git) repo.
3. Push your changes to a branch in your forked repo. 
4. Submit a pull request to souffle-lang/souffle from your forked repo.

> We enforce coding guidelines while running continuous integration framework such as clang-format and linter. 

Reviewer: ```suggestion
Our continuous integration framework enforces coding guidelines with the help of clang-format and clang-tidy. 
```
